### PR TITLE
Feature/ranking page buttons/36

### PR DESCRIPTION
**Pull request by Yin-Lynn-Htun**

### CODE REVIEW

File: src/components/UI/Button/ButtonSecondary.module.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    align-items: center;
+    padding: 8px 30px;
+    width: 11rem;
+
+    background: #ffffff;
+    border: 2px solid #6ac297;
+    box-shadow: 0px 4px 20px rgba(0, 0, 0, 0.1);
+    border-radius: 10px;
+
+    font-size: 1.5rem;
```
Comment (hatoriMain) [inline src/components/UI/Button/ButtonSecondary.module.css:14]: font-size: 2.3rem

File: src/components/UI/Button/ButtonSecondary.module.css [hunk 2]
```diff
@@ -0,0 +1,20 @@
+.button {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    padding: 8px 30px;
+    width: 11rem;
```
Comment (hatoriMain) [inline src/components/UI/Button/ButtonSecondary.module.css:7]: width:15rem
height: 5rem

File: src/components/UI/Button/ButtonSecondary.module.css [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    box-shadow: 0px 4px 20px rgba(0, 0, 0, 0.1);
+    border-radius: 10px;
+
+    font-size: 1.5rem;
+    font-weight: 600;
+    cursor: pointer;
+    font-family: inherit;
+    color: #67c198;
+    text-shadow: 0px 2px 4px rgba(0, 0, 0, 0.1);
+}
```
Comment (hatoriMain) [inline src/components/UI/Button/ButtonSecondary.module.css:20]: Let the button have a gradient Text.(2 changes of file)
change②

```css
.gradientText {
    background: -webkit-linear-gradient(#A3DB95, #67C198);
    -webkit-background-clip: text;
    -webkit-text-fill-color: transparent;
}
```

File: src/components/UI/Button/ButtonSecondary.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import classes from './ButtonSecondary.module.css'
+
+const ButtonSecondary = (props) => {
+    return (
+        <button {...props} className={classes.button}>
+            {props.children}
```
Comment (hatoriMain) [inline src/components/UI/Button/ButtonSecondary.js:7]: Let the button have a gradient Text.(2 changes of file)
change①
```javascript
const ButtonSecondary = (props) => {
    return (
        <button {...props} className={classes.button}>
            <div className={classes.gradientText}> {props.children}</div>
        </button>
    )
}
```

**Review by hatoriMain — CHANGES_REQUESTED** [review]
The button looks very well! 
I commented on some tiny changes to make it looks better. 
1. The size of the font of a button and the button itself.
2. Enable the color of the font to be linear gradient.

**Comment by Yin-Lynn-Htun** [comment]
I changed all the requests, and I also changed the width, height, and font size of the primary button.

**Pull request merged**